docs/guides/websockets/authentication.md

```diff
@@ -0,0 +1,120 @@
+# WebSocket Authentication
+
+Authentication is an important part of establishing your persistent connection with a Directus project. 
+
+## Authentication Modes
+
+There are three authentication modes in Directus.
+
+| Mode | Description |
+|---|---|
+| **public** | No socket authentication is required. |
+| **handshake** | No socket authentication required to connect. First message must be an authentication request sent before the timeout. |
+| **strict** | Authentication is required as a URL parameter on the initial connection. |
+
+### Changing Authentication Modes
+
+Your whole project will use the same, single, authentication mode. You cannot use multiple authentication modes in a single project.
+
+By default, the `handshake` authentication mode is used. If self-hosting your project, you may change the mode used by setting the `WEBSOCKETS_REST_AUTH` and `WEBSOCKETS_GRAPHQL_AUTH` environment variables. 
+
+## REST Authentication Flow
+
+### Public Mode
+
+You do not need to authenticate if using a public authentication mode. 
+
+However, you can only perform operations that are available to your public role. If you need access to non-public data, you should follow the flow for the `handshake` authentication mode.
```

Comment:
I think that's what I wrote, but it obviously isn't clear enough. What was your understanding of what I wrote?

Reply:
Interesting on reading it a couple extra times it can indeed be interpreted like that 🤔 However it really reads to me like "You're limited to the public role. if you want to be authenticated use a different authentication mode". It is not directly clear to me from reading that "follow the flow for the `handshake` authentication mode" is to convey you can authenticate after connecting independent of your authentication mode.

Reply:
Cool makes sense - I can clean that up.